workbench-core/example/infrastructure/integration-tests/support/resources/datasets/dataset.ts

```diff
@@ -48,6 +48,26 @@ export default class Dataset extends Resource {
     return new Endpoint(params);
   }
 
+  public async getAllAccess(): Promise<PermissionsResponse> {
+    const response: AxiosResponse = await this._axiosInstance.get(`${this._api}/permissions`);
+    return validateAndParse(PermissionsResponseParser, response.data);
+  }
+
+  public async getAccess(identityType: 'GROUP' | 'USER', identity: string): Promise<PermissionsResponse> {
```

Comment:
I think you can use the `IdentityType` type here instead.

Reply:
yes!